relational_operators/SelectOperator.hpp

```diff
@@ -163,12 +202,15 @@ class SelectOperator : public RelationalOperator {
   const std::vector<attribute_id> simple_selection_;
 
   std::vector<block_id> input_relation_block_ids_;
+  std::vector<std::vector<block_id>> input_relation_block_ids_in_partition_;
 
   // A single workorder is generated for each block of input relation.
   std::vector<block_id>::size_type num_workorders_generated_;
+  std::vector<std::size_t> num_workorders_generated_in_partition_;
```

Comment:
Similar description as ``input_relation_block_ids_in_partition_`` should be helpful.